### PR TITLE
Widescreen 16:10 support + Launcher bugfixes

### PR DESCRIPTION
**Pull request by snouz**
-Widescreen 16:10 support https://github.com/Albeoris/Memoria/issues/291
-Moguri ending FMVs fix https://github.com/Albeoris/Memoria/issues/127
-Patcher is now signed with a key I generated

Launcher:
-added anti-aliasing option https://github.com/Albeoris/Memoria/issues/304
-added moguri options https://github.com/Albeoris/Memoria/issues/307
-fps setter fix
-resolution resetting fixed
-if resolution in settings.ini not recognized, best is selected
-added screen ratio to resolution
-default to 48000 sampling sound
-Fixed intro music not impacted by music volume https://github.com/Albeoris/Memoria/issues/301
-Makes sure memoria.ini's "=" have two spaces for consistencies https://github.com/Albeoris/Memoria/issues/298
-Fixed crash debug mode if no directory https://github.com/Albeoris/Memoria/issues/308

### CODE REVIEW

File: Memoria.Launcher/Memoria/GameSettingsControl.cs [hunk 1]
```diff
@@ -475,28 +396,13 @@ private void LoadSettings()
...
-                    }
-                    else if (CultureInfo.CurrentCulture.TwoLetterISOLanguageName == "ru")
+                    if (CultureInfo.CurrentCulture.TwoLetterISOLanguageName == "ru")
                     {
-                        _downloadMirrors = new[]
-                        {
-                            "https://ff9.ffrtt.ru/rus/FF9RU.exe",
-                            "https://ff9.ffrtt.ru/rus/Memoria.Patcher.exe"
-                        };
+                        _downloadMirrors = ["https://ff9.ffrtt.ru/rus/FF9RU.exe", "https://ff9.ffrtt.ru/rus/Memoria.Patcher.exe"];
```
Comment (SamsamTS) [inline Memoria.Launcher/Memoria/GameSettingsControl.cs:401]: Why this change? Previously those mirrors would never be checked.

Comment (snouz) [reply Memoria.Launcher/Memoria/GameSettingsControl.cs:401]: Well Albeoris is Russian and I saw this code not being used so I supposed it might have been a mirror to allow non-vpn users...

Yeah maybe that's not very useful, we should always direct to the only up to date source

Comment (Tirlititi) [reply Memoria.Launcher/Memoria/GameSettingsControl.cs]: I have not read everything, but these mirrors are the ones that were used prior to the latest release (Jun 11, 2023). I decided at that time to use this Github as a mirror but also to keep those lines in the code, even disabled, to keep their trace.

**Comment by SamsamTS** [comment]
Why remove the movie volume that I added? There is a good reason that it was there. FMVs have only one audio track that combines Music and Sound effects (and Voices with Echo-S) thus warrant a separate volume control.
